cwltool/singularity.py

```diff
@@ -47,7 +47,7 @@ def _singularity_supports_userns() -> bool:
                 stdout=DEVNULL,
                 universal_newlines=True,
             ).communicate(timeout=60)[1]
-            _USERNS = "No valid /bin/sh" in result
+            _USERNS = "No valid /bin/sh" in result or "/bin/sh doesn't exist in container" in result
```

Comment:
@mr-c I was having some trouble figuring out which version the error message changed but I tweaked the check to support both error messages. Would this be sufficient or would you still like to know which version it changed on. Thank you!

Reply:
Also, I removed the `.DS_Store` from the previous commit.